DriverLoader.h

```diff
@@ -40,14 +40,28 @@
 namespace osvr {
 namespace vive {
     struct CouldNotLoadDriverModule : std::runtime_error {
+#if defined(OSVR_LINUX) || defined(OSVR_MACOSX)
```

Comment:
To have these defines, you need to include <osvr/Util/PlatformDefines.h> - but I'd just leave those constructors available on all platforms. Just because only mac and linux use them doesn't mean having them will hurt the Windows build.